src/Microsoft.Security.Utilities.Core/IdentifiableSecrets.cs

```diff
@@ -19,14 +20,18 @@ public static class IdentifiableSecrets
 {
     public static readonly ulong VersionTwoChecksumSeed = ComputeHisV1ChecksumSeed("Default0");
 
-    public const string CommonAnnotatedKeyRegexPattern = "[A-Za-z0-9]{52}JQQJ99[A-Za-z0-9][A-L][A-Za-z0-9]{16}[A-Za-z][A-Za-z0-9]{7}([A-Za-z0-9]{2}==)?";
+    public const string CommonAnnotatedKeyRegexPattern = "[A-Za-z0-9]{52}JQQJ9(9|D)[A-Za-z0-9][A-L][A-Za-z0-9]{16}[A-Za-z][A-Za-z0-9]{7}([A-Za-z0-9]{2}==)?";
```

Comment:
>JQQJ9(9|D) [](http://example.com/codeflow?start=73&length=10)

the new HISv2 regex need to support derived keys?
cc @LingZhou-gh we may need to update the regex for new HISv2 rules